docs/Buttons.md

```diff
@@ -658,26 +819,437 @@ export const PostEdit = () => (
+        comments: {
+            notifications: {
+                deleted: 'Comment deleted |||| %{smart_count} comments deleted',
+                // ...
+            },
+        },
+    },
+};
+```
+
+Alternately, pass a `successMessage` prop:
+
+```jsx
+<DeleteButton successMessage="Comment deleted successfully" />
+```
+
+## `<DeleteWithConfirmButton>`
 
 Delete the current record after a confirm dialog has been accepted. To be used inside a `<Toolbar/>` component.
 
-| Prop               Required | Type                                             | Default                     | Description                                                             |
-|-----------------------------|--------------------------------------------------|-----------------------------|-------------------------------------------------------------------------|
-| `className`        Optional | `string`                                         | -                           | Class name to customize the look and feel of the button element itself  |
-| `label`            Optional | `string`                                         | 'ra.action.delete'          | label or translation message to use                                     |
-| `icon`             Optional | `ReactElement`                                   | `<DeleteIcon>`              | iconElement, e.g. `<CommentIcon />`                                     |
-| `confirmTitle`     Optional | `ReactNode`                                      | 'ra.message.delete_title'   | Title of the confirm dialog                                             |
-| `confirmContent`   Optional | `ReactNode`                                      | 'ra.message.delete_content' | Message or React component to be used as the body of the confirm dialog |
-| `confirmColor`     Optional | <code>'primary' &#124; 'warning'</code>          | 'primary'                   | The color of the confirm dialog's "Confirm" button                      |
-| `redirect`         Optional | <code>string &#124; false &#124; Function</code> | 'list'                      | Custom redirection after success side effect                            |
-| `translateOptions` Optional | `{ id?: string, name?: string }`                 | {}                          | Custom id and name to be used in the confirm dialog's title             |
-| `mutationOptions`  Optional |                                                  | null                        | options for react-query `useMutation` hook                              |
-| `successMessage`   Optional | `string`                                         | 'ra.notification.deleted'   | Lets you customize the success notification message.                                                                                 |
+| Prop               | Required | Type                                             | Default                     | Description                                                             |
```

Comment:
Shouldn't we either repeat the `DeleteButton` details for props or mention they are the same?